src/js/Controllers/TTSController.js

```diff
@@ -16,13 +25,22 @@ class TTSController {
     }
 
     onResetTimeout(appID, methodName) {
```

Comment:
@iCollin Looks like there is `speakID` property name which equals with messageID. It could be gets from `this.speakID`. Maybe we remove params from method ?

Reply:
speakID is the currently spoken TTS, I think a method parameter would be better because the timeout is then forsure correlated with the speak that caused it.

Reply:
@iCollin please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/18/commits/f9521be3543941e1d8cb27eb7eefdccac872b7a7)